### PR TITLE
James 1773 - Add RecipientIsLocal tests

### PR DESCRIPTION
**Pull request by chibenwa**
To do so, I added tests on parts of concerned JamesMailetContext

### CODE REVIEW

File: server/mailet/mailetcontainer-camel/src/test/java/org/apache/james/mailetcontainer/impl/JamesMailetContextTest.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        usersRepository.addUser(USERMAIL, PASSWORD);
+
+        assertThat(testee.isLocalEmail(mailAddress)).isTrue();
+    }
+
+    @Test
+    public void isLocalEmailShouldBeFalseWhenMailIsNull() throws Exception {
+        assertThat(testee.isLocalEmail(null)).isFalse();
+    }
+}
```
Comment (quynhn) [inline server/mailet/mailetcontainer-camel/src/test/java/org/apache/james/mailetcontainer/impl/JamesMailetContextTest.java]: How is about the others methods? There are so many methods on JamesMailetContext beside of isLocal*

Comment (chibenwa) [inline server/mailet/mailetcontainer-camel/src/test/java/org/apache/james/mailetcontainer/impl/JamesMailetContextTest.java]: :-) maybe a bit too much work not related with the ticket ....

**Comment by rouazana** [comment]
:+1:

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**